Library/Formula/passenger.rb

```diff
@@ -23,6 +23,11 @@ def install
     rake "nginx"
     rake "webhelper"
 
+    # Fixes https://github.com/phusion/passenger/issues/1288
+    rm_rf "buildout/libev"
+    rm_rf "buildout/libeio"
+    rm_rf "buildout/cache"
```

Comment:
I am using Rails 4.2.3 and I am getting this error when installing passenger using homebrew. [Command failed with status (2): [cd buildout/libuv/ && make -j2 libuv.la V=...]]. If you could please help it would be nice. I can't seem to get past this error so I was going to try your fix. I am confused as to where you added your code to fix this issue. paasenger.rb?